addons/l10n_fr_fec/wizard/account_fr_fec.py

```diff
@@ -42,14 +42,13 @@ def _get_where_query(self):
         if self.excluded_journal_ids:
             where_params['excluded_journal_ids'] = tuple(self.excluded_journal_ids.ids)
             where_query += "AND am.journal_id NOT IN %(excluded_journal_ids)s\n"
-        return where_query, where_params
+        return SQL(where_query, **where_params)
 
     def _do_query_unaffected_earnings(self):
         ''' Compute the sum of ending balances for all accounts that are of a type that does not bring forward the balance in new fiscal years.
             This is needed because we have to display only one line for the initial balance of all expense/revenue accounts in the FEC.
         '''
-        where_query, where_params = self._get_where_query()
-        sql_query = f'''
+        sql_query = SQL('''
```

Comment:
Keeping this opening quote there makes the identation below weird to me.
That's why I usually put all parameters on the next line, like in
```suggestion
        sql_query = SQL(
            '''
```
But that's just _my_ preference...